etcd-manager/go.mod

```
@@ -3,71 +3,81 @@ module sigs.k8s.io/etcdadm/etcd-manager
 go 1.18
 
 require (
-	cloud.google.com/go/compute v1.9.0
-	github.com/Azure/azure-sdk-for-go v66.0.0+incompatible
+	cloud.google.com/go/compute/metadata v0.2.1
+	github.com/Azure/azure-sdk-for-go v67.1.0+incompatible
 	github.com/Azure/go-autorest/autorest/azure/auth v0.5.11
 	github.com/Azure/go-autorest/autorest/to v0.4.0
-	github.com/aws/aws-sdk-go v1.44.90
+	github.com/aws/aws-sdk-go v1.44.145
 	github.com/blang/semver/v4 v4.0.0
-	github.com/digitalocean/godo v1.83.0
+	github.com/digitalocean/godo v1.91.1
 	github.com/golang/protobuf v1.5.2
-	github.com/gophercloud/gophercloud v1.0.0
-	github.com/hetznercloud/hcloud-go v1.35.2
-	github.com/prometheus/client_golang v1.13.0
-	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.9
+	github.com/gophercloud/gophercloud v1.1.0
+	github.com/hetznercloud/hcloud-go v1.37.0
+	github.com/prometheus/client_golang v1.14.0
+	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.10
 	go.etcd.io/etcd/api/v3 v3.5.4
 	go.etcd.io/etcd/client/v3 v3.5.4
-	golang.org/x/net v0.0.0-20220826154423-83b083e8dc8b
-	golang.org/x/oauth2 v0.0.0-20220822191816-0ebed06d0094
-	google.golang.org/api v0.94.0
-	google.golang.org/grpc v1.48.0
+	golang.org/x/net v0.2.0
+	golang.org/x/oauth2 v0.2.0
+	google.golang.org/api v0.103.0
+	google.golang.org/grpc v1.50.1
 	gopkg.in/gcfg.v1 v1.2.3
-	k8s.io/apimachinery v0.25.0
-	k8s.io/client-go v0.25.0
-	k8s.io/klog/v2 v2.80.0
-	k8s.io/kops v1.25.0-beta.1.0.20220902124005-323035fdfce5
-	k8s.io/mount-utils v0.25.0
-	k8s.io/utils v0.0.0-20220823124924-e9cbc92d1a73
+	k8s.io/apimachinery v0.25.4
+	k8s.io/client-go v0.25.4
+	k8s.io/klog/v2 v2.80.1
+	k8s.io/kops v1.26.0-alpha.2.0.20221129070522-374daea6dad2
```

Is there any reason for not using the tagged version? It is quite recent.
```suggestion
	k8s.io/kops v1.26.0-alpha.2
```